### DISCUSSION TITLE
Install failing on AWS EC2's for both Debian 9/11 and Ubuntu Images

### DISCUSSION THREAD

**Discussion by yamasaki760** [Ysurac/openmptcprouter#2515] [General]
Hello, I've got my WRT3200ACM flashed with OpenMPTcpRouter successfully, now I'm trying to set up my VPS in AWS.  However, I can't seem to get past the install process.  I always seem to run into this error:

```
E: Failed to fetch https://repo.openmptcprouter.com/dists/buster/main/binary-amd64/Packages.gz  File has unexpected size (10947 != 10948). Mirror sync in progress? [IP: 37.19.207.34 443]
   Hashes of expected file:
    - Filesize:10948 [weak]
    - SHA256:0eebcf2c48eff7681dba84708d965fc4580a4ae90c3da76776e8c2619024d9ed
    - SHA1:075d3ada93d4f94f6f5cd541ea0396b0b84cdce6 [weak]
    - MD5Sum:15c873b7a94e9e350431f824fdbbfe39 [weak]
   Release file created at: Mon, 22 Aug 2022 06:45:16 +0000
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

Ive tried GCP VM's and AWS EC2's both in N. Virginia with similar results. 

Thanks in advance!

**Comment by jaymwages** [comment]
I filed this on the VPS repo yesterday and it was resolved. I was able to successfully execute the VPS script this morning.

https://github.com/Ysurac/openmptcprouter-vps/issues/69

**Reply by yamasaki760** [reply]
Thank you, Ive verified also!  working on Ubuntu 20.04
